cloud/lxc/lxc_container.py

```diff
@@ -448,7 +448,7 @@
         'zfs_root'
     ],
     'btrfs': [
-        'lv_name', 'vg_name', 'thinpool', 'zfs_root'
+        'lv_name', 'vg_name', 'thinpool', 'zfs_root', 'fs_type', 'fs_size'
```

Comment:
Because without this fix i catch error like this
"err": "filesystem type and size are only valid with block devices\n"